Marlin/src/HAL/HAL_LPC1768/fast_pwm.cpp

```diff
@@ -24,7 +24,7 @@
 
 #include "../../inc/MarlinConfigPre.h"
 
-#if ENABLED(FAST_PWM_FAN)
+#if ANY(FAST_PWM_FAN, SPINDLE_LASER_PWM) //FAST_PWM_FAN Shouldn't be dependency for pwm control; but sanguino_atmega1284p won't build this file; means hal is borkened
```

Comment:
If this file won't build for Sanguino, it may be due to timer differences on the 1284p. And that issue may now be patched.

Reply:
I changed the note with e26b33ed5c3516ce41570a9254899546ff13431f which also added a NEEDS_PWM meta note